.github/workflows/test.yml

```diff
@@ -16,29 +16,30 @@ jobs:
     name: Calculate changes
     runs-on: ubuntu-latest
     outputs:
-      src-esp32: ${{ steps.changes.outputs.src-esp32 }}
-      src-native: ${{ steps.changes.outputs.src-native }}
-      src-common: ${{ steps.changes.outputs.src-common }}
-      src-servers: ${{ steps.changes.outputs.src-servers }}
-      src-cli: ${{ steps.changes.outputs.src-cli }}
+      src-esp32: ${{ steps.filter.outputs.src-esp32 }}
+      src-native: ${{ steps.filter.outputs.src-native }}
+      src-common: ${{ steps.filter.outputs.src-common }}
+      src-servers: ${{ steps.filter.outputs.src-servers }}
+      src-cli: ${{ steps.filter.outputs.src-cli }}
+      src-ffi: ${{ steps.filter.outputs.src-ffi }}
     container:
       image: ghcr.io/viamrobotics/micro-rdk-dev-env:amd64
     steps:
     - name : Checkout main branch code
       if: github.event_name != 'pull_request_target'
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       with:
         fetch-depth: 2
     - name: Check out PR branch code
       if: github.event_name == 'pull_request_target'
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       with:
         ref: ${{ github.event.pull_request.head.sha }}
         fetch-depth: 2
     - run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
     - name: path filter
-      uses: dorny/paths-filter@v2
-      id: changes
+      uses: dorny/paths-filter@v3
+      id: filter
```

Comment:
What is the effect of this?
The example [here](https://github.com/dorny/paths-filter?tab=readme-ov-file#example) still uses `id: changes` for v3

Reply:
that just change the key to access the output so `steps.filter` instead of  `steps.changes`